src/main/java/pl/betse/beontime/timesheet/model/MonthDayBody.java

```diff
@@ -0,0 +1,38 @@
+package pl.betse.beontime.timesheet.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.validator.constraints.Length;
+
+import pl.betse.beontime.timesheet.validation.CreateTimeEntry;
+
+import javax.validation.constraints.Max;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.PositiveOrZero;
+import java.math.BigDecimal;
+
+@Builder
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class MonthDayBody {
+    @NotEmpty(groups = {CreateTimeEntry.class}, message = "Date can't be empty!")
+    @NotNull(groups = {CreateTimeEntry.class}, message = "Date can't be null!")
+    private String date;
```

Comment:
why date is sent as a String? Shouldn't it be a LocalDate with given format?

Reply:
Jackson have problems with parsing LocalDate...